synapse/lib/layer.py

```diff
@@ -1451,25 +1451,29 @@ async def _layrV5toV6(self):
         self.meta.set('version', 6)
         self.layrvers = 6
 
-        logger.warning(f'...complete!')
+        logger.warning('...complete!')
 
     async def _initLayerStorage(self):
 
         slabopts = {
             'readonly': self.readonly,
             'max_dbs': 128,
-            'map_async': True,
             'readahead': True,
             'lockmemory': self.lockmemory,
-            'growsize': self.growsize,
+            'map_async': self.mapasync,
```

Comment:
Should ``self.maxreplaylog`` be mapped in here as ``max_replay_log`` ?

Reply:
yes. sorry forgot to mark PR as WIP